maakay-bot.py

```diff
@@ -617,6 +618,33 @@ async def tournament_reward(ctx, tournament_id: str, user: discord.Member):
         embed.add_field(name="Sorry!", value="You donot have correct permission to reward this challenge.")
         await ctx.send(embed=embed, hidden=True)
 
+
+@slash.subcommand(base="tournament", name="all", description="list all the active challenges!!")
+async def tournament_all(ctx):
+    
+    await ctx.defer(hidden=True)
+    print(ctx.author)
+    discord_user, created = await sync_to_async(User.objects.get_or_create)(discord_id=str(ctx.author.id))
+
+    embed = discord.Embed(title=f"{str(ctx.author)[0:-5]}'s Active Tournaments")
+
+    if Tournament.objects.filter(Q(hosted_by=discord_user) | Q(winner=discord_user), Q(status=Tournament.ONGOING)).exists():
```

Comment:
could we remove the filter of the winner since once the winner is selected, the challenge is turned into the status of "Tournament.COMPLETED".
`if Tournament.objects.filter(Q(hosted_by=discord_user), Q(status=Tournament.ONGOING)).exists():`